.github/workflows/rust.yml

```diff
@@ -18,7 +18,7 @@ jobs:
   run_checks:
     strategy:
       matrix:
-        rust_toolchain_version: [1.67, 1.68, 1.69]
+        rust_toolchain_version: [1.71.1]
```

Comment:
I think we want to keep compatibility with the other versions for a while. Can you add this one instead of replacing the others?